apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -152,6 +152,32 @@ export default function GlobalStyle () {
                   cursor: pointer;
                 }
               }
+
+              div.scroll-container {
```

Comment:
I think it's not a part of the common components.
@Dimitreee should we put it close to the FullScreen component?

Reply:
Is it possible inject it in FullscreenWrapper?

Reply:
Use css in js, please. Place this components near FullScreen. see apps/condo/domains/common/components/LinkWithIcon.tsx